### ISSUE TITLE
integration with jupyter lab support datascience workflow!!!

### ISSUE THREAD

**Issue by hyperleoon** [DTStack/molecule#554]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by wewoor** [comment]
@hyperleoon Yeah, Thanks for your support!!, the **jupyter** is a very important part of our product, we also think about it.
